zaza/openstack/charm_tests/magnum/tests.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python3
+#
+# Copyright 2019 Canonical Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Encapsulate magnum testing."""
+import logging
+import json
+import os
+import subprocess
+from urllib import parse as urlparse
+
+import zaza.model
+import zaza.openstack.charm_tests.nova.utils as nova_utils
+import zaza.openstack.charm_tests.test_utils as test_utils
+import zaza.openstack.utilities.openstack as openstack_utils
+import zaza.charm_lifecycle.utils as charm_lifecycle_utils
+
+# Resource and name constants
+IMAGE_NAME = 'fedora-coreos'
```

Comment:
Have the image name as a global variable in the setup file, and import it here. Don't define it in two places.